.circleci/config.yml

```diff
@@ -111,7 +111,7 @@ workflows:
             parameters:
               image-tag:
                 - 1.16.0-erlang-26.2.1-alpine-3.18.4
```

Comment:
Wasn't the issue in 3.18.4?
Do we need to update this to: [1.16.2-erlang-25.3.2.11-alpine-3.18.6](https://hub.docker.com/layers/hexpm/elixir/1.16.2-erlang-25.3.2.11-alpine-3.18.6/images/sha256-a4b03a1648cc9a8d2a6596247f042e4b00f7678e7aa354cc11b822afc171fe76?context=explore) ?

Reply:
If I just change those from 3.18.4 to 3.18.6 it says the image isn't found.

`Error response from daemon: manifest for hexpm/elixir:1.16.0-erlang-26.2.1-alpine-3.18.6 not found: manifest unknown: manifest unknown`

I'm a little confused by the number of different image tags this has. Are they all needed? Can we just keep the one 3.18.6 that works?

Reply:
- `1.16.2-erlang-26.2.4-alpine-3.19.1`
- `1.15.7-erlang-26.2.4-alpine-3.19.1`
- `1.14.5-erlang-25.3.2.11-alpine-3.19.1`

Those should all work and are the latest published patch versions for Elixir, Erlang, and Alpine.

> I'm a little confused by the number of different image tags this has. Are they all needed? Can we just keep the one 3.18.6 that works?

We test with multiple images to ensure compatibility with the Elixir/Erlang versions we're currently supporting.

Reply:
Thanks, that looks good. Resubmitting for approval.